packages/react-vapor/src/components/select/tests/MultiSelectConnected.spec.tsx

```diff
@@ -68,11 +68,21 @@ describe('Select', () => {
             // open the dropdown
             userEvent.click(screen.getByRole('button', {name: /select an option/i}));
 
-            const lists = screen.getAllByRole('list');
-
-            expect(within(lists[1]).getByText(':seed:')).toBeVisible();
-            expect(within(lists[1]).getByText(':potato:')).toBeVisible();
-            expect(within(lists[1]).getByText('🍟')).toBeVisible();
+            expect(
+                screen.getByRole('option', {
+                    name: /:seed:/i,
+                })
+            ).toBeVisible();
+            expect(
+                screen.getByRole('option', {
+                    name: /:seed:/i,

```

Comment:
I think it would be :potato:?
```suggestion
                    name: /:potato:/i,

```